resources/lang/en/people.php

```diff
@@ -10,28 +10,28 @@
 
     //index
     'people_not_found' => 'Contact not found',
-    'people_list_number_kids' => '1 kid|:count kids',
+    'people_list_number_kids' => 'One child|:count children',
     'people_list_last_updated' => 'Last consulted:',
-    'people_list_number_reminders' => '1 reminder|:count reminders',
+    'people_list_number_reminders' => 'One reminder|:count reminders',
     'people_list_blank_title' => 'You don’t have anyone in your account yet',
     'people_list_blank_cta' => 'Add someone',
     'people_list_sort' => 'Sort',
-    'people_list_stats' => '1 contact|:count contacts',
+    'people_list_stats' => 'One contact|:count contacts',
```

Comment:
I don't see any reason not to keep the number even for "1"
```suggestion
    'people_list_stats' => '1 contact|:count contacts',
```